runtime/bcutil/attrlookup.gperf

```diff
@@ -24,7 +24,10 @@
  * This file is the input to the gperf utility for generating a perfect
  * hash table of attribute names, which is used by attributeTagFor().
  *
- * To generate attrlookup.h, use the following command-line with gperf 3.0.4:
+ * Types generated by gperf cause compiler issues in the OpenJ9 Windows builds,
+ * use the existing types.
+ *
```

Comment:
If things can be made to work using `size_t`, this comment is inappropriate.

Reply:
Thanks! I'm still trying to verify I think I've been running valhalla personal builds wrong and so far took 3 hours to compile today :/